documentation/src/main/asciidoc/user_guide/persistence_storemigrator.adoc

```diff
@@ -0,0 +1,117 @@
+=== Store Migrator
+Infinispan 9.0 introduced non-backwards compatible changes to Infinispan's internal marshalling. Consequently, cache stores
+created with prior versions of Infinispan can no longer be read by Infinispan 9.x. Furthermore, some previous store
+implementations, such as the JDBC Mixed and Binary stores, are no longer provided. Therefore, in order to assist users,
+we have created a migration tool `StoreMigrator.java`, that reads data from Infinispan 8.2.x stores and rewrites their
+content using the latest marshalling implementation.
+
+==== Usage
+The`StoreMigrator` takes a single argument, the path to a
+`.properties` file which must contain the configuration properties for both the source and target stores. An example
+properties file containing all applicable configuration options can be found
+link:https://github.com/infinispan/infinispan/blob/master/tools/src/main/resources/migrator.properties[here].
+
+To use the migrator, you need the `infinispan-tools-{infinispanversion}.jar` as well as any dependencies (e.g. JDBC drivers)
+required by your source and target databases on your classpath. An example maven pom, that will execute the migrator
+via `mvn exec:java` is presented below:
+
+[source,xml]
+----
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.infinispan.example</groupId>
+    <artifactId>jdbc-migrator-example</artifactId>
+    <version>1.0-SNAPSHOT</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.infinispan</groupId>
+            <artifactId>infinispan-tools</artifactId>
+            <version>9.3.0-SNAPSHOT</version>
```

Comment:
Use property replacement to use the version

Reply:
I tried with `{infinispanversion}` but it does not seem to work when in a `[source,xml]` block